src/commcare_cloud/commands/deploy.py

```diff
@@ -46,6 +46,10 @@ class Deploy(CommandBase):
     )
 
     def modify_parser(self):
+        if len(sys.argv) <= 1:
+            # No environment specified, so no need to add environment-specific repositories
```

Comment:
surely an environment is required?

Reply:
The next line (`sys.argv[1]`) fails if an environment is not specified.

Reply:
@millerdev is that an answer to my question? With this change what will happen if you don't specify an env?

Reply:
I didn't really understand your first question. With this change you will see a help message rather than a traceback with `IndexError: list index out of range`

Reply:
I could have included more context -- the rationale was that in the process of mentoring, just to see if we had installed `commcare-cloud` correctly, I had the mentee run `cchq` with no args. When it printed out a traceback, it made both of us think that we had installed it incorrectly. Commcare-cloud should handle this situation gracefully.  This line was what was causing the traceback -- if the user specifies no arguments, it's not trying to run anything, it's just trying to figure out what commands are legal and then respond with a 'missing args' message.

Reply:
Thanks. It's just not at all obvious from this code change that returning `None` will result in the help text being shown.